src/ragger/backend/ledgercomm.py

```diff
@@ -71,6 +73,7 @@ def __exit__(self, *args, **kwargs):
 
     def send_raw(self, data: bytes = b"") -> None:
         logger.debug("Sending '%s'", data)
+        self.apdu_logger.debug("=> %s", data.hex())
```

Comment:
Rather than multiplying the number of loggers, you could define several handler for `apdu_logger`, one on the `stdout` (which would then take place of the previous `logger`) and the second, optional, which would log into a file.

Reply:
Done